testing/run-scripts/throughput.py

```diff
@@ -8,9 +8,11 @@
 import time
 import urllib.parse
 
-FLOOD_SIZE_MB = 64
+FLOOD_SIZE_MB = 32
 FLOOD_MAX_SPEED_MB = 5
 
+LATENCY_MS = 150
```

Comment:
Mind putting a link into that wonderful comment you made in the PR conversation here?  So that we know why 150ms was chosen.